### PR TITLE
Fixed one small issue for BootstrapTable.js

### PR DESCRIPTION
**Pull request by athenaxing**
int componentWillReceiveProps functions

Add checking code
   if (nextProps.data === undefined) {
      return;
    }
Before 
   this.store.setData(nextProps.data.slice());

### CODE REVIEW

File: src/BootstrapTable.js [hunk 1]
```diff
@@ -215,6 +215,9 @@ class BootstrapTable extends Component {
     let { replace } = nextProps;
     replace = replace || this.props.replace;
 
+    if (nextProps.data === undefined) {
```
Comment (AllenFang) [inline src/BootstrapTable.js]: try following instead
if (!nextProps.data) {
  return;
}

**Comment by athenaxing** [comment]
Thank you.




我思故我在
On 07/08/2017 17:58, Allen wrote:

@AllenFang requested changes on this pull request.

In src/BootstrapTable.js:

> @@ -215,6 +215,9 @@ class BootstrapTable extends Component {
     let { replace } = nextProps;
     replace = replace || this.props.replace;
 
+    if (nextProps.data === undefined) {


try following instead
if (!nextProps.data) {
return;
}

File: src/BootstrapTable.js (hunk 1 above)
Comment (athenaxing) [inline src/BootstrapTable.js]: Have changed the code to 
if (!nextProps.data) {
  return;
}

**Comment by AllenFang** [comment]
Release on `v3.4.5` thanks your contribution

**Pull request merged**